store/sqlstore/file_info_store.go

```diff
@@ -734,3 +734,20 @@ func (fs SqlFileInfoStore) GetFilesBatchForIndexing(startTime int64, startFileID
 
 	return files, nil
 }
+
+func (fs SqlFileInfoStore) GetStorageUsage(allowFromCache, includeDeleted bool) (int64, error) {
```

Comment:
Why pass `allowFromCache` here?

Reply:
Our struct embedding layer code needs it.